assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceTilesWidget/index.stories.js

```diff
@@ -278,8 +278,8 @@ TwoTilesWithZeroTile.args = {
 			.dispatch( MODULES_ANALYTICS_4 )
 			.receiveResourceDataAvailabilityDates( {
 				audience: {
-					'properties/12345/audiences/1': audienceDate,
-					'properties/12345/audiences/4': dataAvailabilityDate,
+					'properties/12345/audiences/1': dataAvailabilityDate,
```

Comment:
We should show the first tile as zero tile for better visibility on all screens.